coremltools/converters/mil/frontend/torch/torch_op_registry.py

```diff
@@ -127,3 +129,14 @@ def func_wrapper(func):
         # decorator called without argument
         return func_wrapper
     return func_wrapper(_func)
+
+
+def is_torch_fx_node_supported(torch_fx_node: torch.fx.Node) -> bool:
+    if torch_fx_node.op != "call_function":
```

Comment:
Yeah, I'm confused by `"call_function"`. What is that?

Reply:
Added a comment to explain
```
    # There are 5 types of torch fx node:
    #     1. placeholder
    #     2. get_attr
    #     3. call_function
    #     4. call_module
    #     5. output
    # Only "call_function" contains a single PyTorch op that can be queried in _TORCH_OPS_REGISTRY
```